strr-web/stores/account.ts

```diff
@@ -71,17 +71,21 @@ export const useBcrosAccount = defineStore('bcros/account', () => {
       })
   }
 
-  async function acceptTos (acceptance: boolean, versionId?: string): Promise<TermsOfServiceI | void> {
-    return await axiosInstance.patch<TermsOfServiceI>(`${strrApiURL}/account`,
+  async function acceptTos (acceptance: boolean, versionId: string): Promise<TermsOfServiceI | void> {
+    return await axiosInstance.patch<TermsOfServiceI>(`${strrApiURL}/users/tos`,
       {
-        acceptTermsAndConditions: acceptance,
-        termsVersion: versionId
+        istermsaccepted: acceptance,
```

Comment:
No came case?

Reply:
Ask Lekshmi: get 500 error if I do not do lowercase

Reply:
ok, I assume you and Lekshmi are ok with lowercase. Resolved the comment.